Pipfile

```diff
@@ -4,12 +4,14 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+"psycopg2-binary" = "*"
 boto3 = "*"
 dj-database-url = "*"
-Django = "==1.11.23"
+Django = "==1.11.27"
```

Comment:
Can you bump it to 1.11.28 to cover the last scurity update? https://docs.djangoproject.com/en/dev/releases/1.11.28/

Reply:
I most certainly can.